examples/web/frp.html

```diff
@@ -0,0 +1,121 @@
+<html>
+    <head>
+        <meta charset="UTF-8"></meta>
+        <title>FRP</title>
+        <script type="text/javascript" src="../../dist/js-csp.min.js"></script>
+        <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.1/jquery.min.js"></script>
+    </head>
+    <body>
+        <style>
+         #display.waiting {
+             color: gray;
+         }
+        </style>
+        <input type="text" name="" value="" id="input" />
+        <div id="output"></div>
+        <script type="text/javascript">
+        
+        function fromEvent($element, eventName) {
+          var outc = csp.chan();
+          csp.go(function* () {
+            $element.on(eventName, evt => { yield csp.put(outc, evt) });
```

Comment:
invalid use of yield in inner function, not sure if this will work?
`$element.on(eventName, evt => csp.putAsync(outc, evt));`